src/appleseed/renderer/kernel/lighting/subsurfacesampler.h

```diff
@@ -194,18 +194,6 @@ void SubsurfaceSampler::sample(
         if (incoming_point.get_material() == outgoing_material ||
             incoming_point.get_opposite_material() == outgoing_material)
         {
-#ifdef APPLESEED_WITH_OSL
```

Comment:
Why is it not needed here, but needed in `pathtracer.h`? I'm not sure to know the rationale...